app/models/presenter/auction.rb

```diff
@@ -0,0 +1,196 @@
+# require 'action_view'
```

Comment:
I think this is a duplicate file from a rebase after Jessie moving things around in #506. It can be deleted entirely.